pkgs/tools/security/minio-certgen/default.nix

```
@@ -2,16 +2,16 @@
 
 buildGoModule rec {
   pname = "minio-certgen";
-  version = "0.0.2";
+  version = "1.0.1";
 
   src = fetchFromGitHub {
     owner = "minio";
     repo = "certgen";
     rev = "v${version}";
-    sha256 = "sha256-HtzcoEUMt3LpQNyT0wGcmc4Q70QqHx7QpjrDh4YSO/Q=";
+    sha256 = "sha256-YLFrW0w1H6u/lTP4fWPcRFTG9gIm228J2KcE4VSeG2Q=";
   };
 
-  vendorSha256 = "sha256-pQpattmS9VmO3ZIQUFn66az8GSmB4IvYhTTCFn6SUmo=";
+  vendorSha256 = null;
```

```suggestion
  vendorSha256 = null;
```
vendorSha256 = "sha256-pQpattmS9VmO3ZIQUFn66az8GSmB4IvYhTTCFn6SUmo=";

How to fix this?

---

Maybe is because the project does not rely on in anything more than stdlib of Go.

Yet I think is safer to use the generated sha256, as leaving it null give me strange errors in build:

```
these 2 derivations will be built:
  /nix/store/6bvdnrsv7207m7ilyvid9c80ipxvg7l9-source.drv
  /nix/store/y0z8w4ml9qhrs7y07vrvdbsnbdb9yhcy-minio-certgen-1.0.1.drv
building '/nix/store/6bvdnrsv7207m7ilyvid9c80ipxvg7l9-source.drv'...
error checking the existence of https://tarballs.nixos.org/sha256/:
curl: (6) Could not resolve host: tarballs.nixos.org

trying https://github.com/minio/certgen/archive/v1.0.1.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0curl: (6) Could not resolve host: github.com
Warning: Problem : timeout. Will retry in 1 seconds. 3 retries left.
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0curl: (6) Could not resolve host: github.com
Warning: Problem : timeout. Will retry in 2 seconds. 2 retries left.
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0curl: (6) Could not resolve host: github.com
Warning: Problem : timeout. Will retry in 4 seconds. 1 retries left.
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0curl: (6) Could not resolve host: github.com
error: cannot download source from any mirror
error: builder for '/nix/store/6bvdnrsv7207m7ilyvid9c80ipxvg7l9-source.drv' failed with exit code 1;
       last 10 log lines:
       >   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
       >                                  Dload  Upload   Total   Spent    Left  Speed
       >   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0curl: (6) Could not resolve host: github.com
       > Warning: Problem : timeout. Will retry in 1 seconds. 3 retries left.
       >   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0curl: (6) Could not resolve host: github.com
       > Warning: Problem : timeout. Will retry in 2 seconds. 2 retries left.
       >   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0curl: (6) Could not resolve host: github.com
       > Warning: Problem : timeout. Will retry in 4 seconds. 1 retries left.
       >   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0curl: (6) Could not resolve host: github.com
       > error: cannot download source from any mirror
       For full logs, run 'nix log /nix/store/6bvdnrsv7207m7ilyvid9c80ipxvg7l9-source.drv'.
error: 1 dependencies of derivation '/nix/store/y0z8w4ml9qhrs7y07vrvdbsnbdb9yhcy-minio-certgen-1.0.1.drv' failed to build
```

---

Probably because the project does not rely on in anything more than stdlib of Go.